pkg/core/blockchain_test.go

```diff
@@ -2003,3 +2004,157 @@ func setSigner(tx *transaction.Transaction, h util.Uint160) {
 		Scopes:  transaction.Global,
 	}}
 }
+
+func TestBlockchain_StartFromExistingDB(t *testing.T) {
+	var stores = map[string]func(testing.TB, string) (storage.Store, string){
+		"BoltPS":  newBoltStoreForTestingWithPath,
+		"LevelPS": newLevelDBForTestingWithPath,
```

Comment:
Do we need to test two DBs here? The logic we test is a layer above all of this, any DB backend should work the same.

Reply:
Right, fixed.